pkg/volume/csi/csi_test.go

```diff
@@ -562,6 +564,7 @@ func TestCSI_VolumeAll(t *testing.T) {
 				}
 				csiDetacher := volDetacher.(*csiAttacher)
 				csiDetacher.csiClient = csiClient
+				<-waitForAttachDone
```

Comment:
does the WaitForAttach call on line 322 not guarantee the attach was done by the time we got here?

Reply:
Yes, that's better. Just first I'am trying to solve `Detach` problem so I move it to line 567. Now I moved it to line 322.